src/app/services/http/transmart-http.service.ts

```diff
@@ -155,7 +156,7 @@ export class TransmartHttpService {
    */
   getCountsPerStudy(constraint: Constraint): Observable<object> {
     const urlPart = 'observations/counts_per_study';
-    const body = {constraint: TransmartConstraintMapper.mapConstraint(constraint)};
+    const body = {constraint: TransmartConstraintMapper.mapConstraintOnPatientLevel(constraint)};
```

Comment:
still not quite get the motivation behind this change, we replace mapConstraint with mapConstraintOnPatientLevel everywhere, this means mapConstraint no longer exists, right? If this is just a simple renaming of the function, I am ok with it, just seems that the new name does not make the purpose of the function clearer, maybe also add some comments explaining what the function actually does.

Reply:
`mapConstraint` is still used for some calls, e.g. in gb-backend-service for the /queries calls, also for the transmart-packer export. `mapConstraintOnPatientLevel` means map constraint as before, but additionally wrap it with patient subselection. Not sure how to rename it.

Reply:
I see, it's clear, then I think it is fine like this 👍 


ps: for now